### PR TITLE
Addressed the current sonar issues.

### PR DESCRIPTION
**Pull request by jepp3**
Just a small cleanup. We have a A rating already, but good to remove the minor issues we have.

### CODE REVIEW

File: impl/src/main/java/slack/ThreadMessageHandler.java [hunk 1]
```diff
@@ -1,11 +1,6 @@
 package slack;
 
-import api.Answer;
-import api.AnswerResource;
-import api.Question;
-import api.QuestionResource;
-import api.User;
-import api.UserResource;
+import api.*;
```
Comment (flowertwig) [inline impl/src/main/java/slack/ThreadMessageHandler.java]: Do we have a sonar rule for this?

Comment (jepp3) [reply impl/src/main/java/slack/ThreadMessageHandler.java]: no, that was probably my editor config at my mac that did something.

File: impl/src/main/java/slack/ThreadMessageHandler.java [hunk 2]
```diff
@@ -62,7 +62,7 @@ public boolean shouldHandle(String type, JsonObject body) {
                 //No thread found -> create new
                 .onErrorResumeNext(throwable -> {
                     if (throwable instanceof WebException
-                            && NOT_FOUND.equals(((WebException) throwable).getStatus())) {
+                        && NOT_FOUND.equals(((WebException)throwable).getStatus())) {
```
Comment (splitfeed) [inline impl/src/main/java/slack/ThreadMessageHandler.java]: This linebreak is still horrible though 😄

File: impl/src/main/java/slack/ThreadMessageHandler.java [hunk 3]
```diff
@@ -144,11 +145,12 @@ private Auth as(Long userId) {
...
-        ));
+            ));
     }
 
     /**
-     * TODO figure out how to create a title from the message
+     * Gets the title from the message.
+     *
      * @param message the message from slack
      * @return the title
```
Comment (splitfeed) [inline impl/src/main/java/slack/ThreadMessageHandler.java]: No need for double descriptions like "Gets the title" and "@return The title", just a @return is good enough when it describes the purpose :)

Comment (jepp3) [reply impl/src/main/java/slack/ThreadMessageHandler.java]: fixed :)

File: impl/src/main/java/slack/ThreadMessageHandler.java (hunk 2 above)
Comment (jepp3) [reply impl/src/main/java/slack/ThreadMessageHandler.java]: fixed :)

File: impl/src/main/java/slack/ThreadMessageHandler.java [hunk 4]
```diff
@@ -61,10 +61,10 @@ public boolean shouldHandle(String type, JsonObject body) {
 
                 //No thread found -> create new
                 .onErrorResumeNext(throwable -> {
-                    if (throwable instanceof WebException
-                        && NOT_FOUND.equals(((WebException)throwable).getStatus())) {
-                        return createQuestionAndPostToSlack(message);
-
+                    if (throwable instanceof WebException) {
```
Comment (flowertwig) [inline impl/src/main/java/slack/ThreadMessageHandler.java]: There are sonar rules for not nesting ifs, but I guess we're not using them. 😄

Comment (jepp3) [reply impl/src/main/java/slack/ThreadMessageHandler.java]: fixed

**Comment by jepp3** [comment]
@kindanice the code coverage is the files modified is less than the accepted level. But it should not stop us from merging this one.

**Pull request merged**